sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -161,7 +164,260 @@ def plot(self, *, include_values=True, cmap='viridis',
         self.ax_ = ax
         return self
 
+    @classmethod
+    def from_estimator(
+        cls,
+        estimator,
+        X,
+        y,
+        *,
+        labels=None,
+        sample_weight=None,
+        normalize=None,
+        display_labels=None,
+        include_values=True,
+        xticks_rotation="horizontal",
+        values_format=None,
+        cmap="viridis",
+        ax=None,
+        colorbar=True,
+    ):
+        """Plot Confusion Matrix given an estimator and some data.
+
+        Read more in the :ref:`User Guide <confusion_matrix>`.
 
+        .. versionadded:: 0.24
+
+        Parameters
+        ----------
+        estimator : estimator instance
+            Fitted classifier or a fitted :class:`~sklearn.pipeline.Pipeline`
+            in which the last estimator is a classifier.
+
+        X : {array-like, sparse matrix} of shape (n_samples, n_features)
```

Comment:
doesn't `array-like`? include `sparse matrix`?

Reply:
nop

Reply:
Now we are almost up-to-date: https://scikit-learn.org/dev/developers/contributing.html#guidelines-for-writing-documentation :)